examples/00-EDB/14_edb_create_parametrized_design.py

```diff
@@ -2,39 +2,39 @@
 EDB: parameterized design
 ------------------------
 This example shows how to
-1, Create an HFSS simulation project using SimulationConfiguration class.
+1, Set up an HFSS project using SimulationConfiguration class.
 2, Create automatically parametrized design.
 """
 ######################################################################
+# The following layout will be created in this example
```

Comment:
```suggestion
# This image shows the layout created in this example:
```
There was content earlier that was still formatted like what was here. I changed it to "View expected project" but you will want to standardize.